bgpd/bgp_srv6.c

```diff
@@ -83,6 +83,9 @@ void bgp_srv6_unicast_sid_endpoint(struct bgp *bgp, afi_t afi,
 	struct seg6local_context ctx = {};
 	struct in6_addr *unicast_sid_ls = NULL;
 
+	if (afi == AFI_UNSPEC)
```

Comment:
In which scenarios are these functions called with AFI_UNSPEC?

Reply:
today, in bgp_zebra_srv6_sid_notify(), if DT46 sid is received, this may call sid_unicast_withdraw() when afi is AFI_UNSPEC. Actually, I did replace that patch with a more relevant patch that targets this.